welcome.md

```diff
@@ -4,30 +4,28 @@ sd_hide_title: true
 
 # Home
 
-{doc}`PyMC <pymc:index>` is a probabilistic programming library for Python that allows users to build Bayesian models with a simple Python API and fit them using Markov chain Monte Carlo (MCMC) methods.
-PyMC focuses on usability, flexibility, scalability, and extensibility.
-Along with core model specification and fitting functionality,
-PyMC integrates with {doc}`ArviZ <arviz:index>` for exploratory analysis of the results.
+<img src="https://github.com/pymc-devs/brand/blob/main/logos/1-pymcdevs.png?raw=true"/>
```

Comment:
I personally don't like the logo here very much, but happy to do whatever most people prefer

Reply:
For me something is missing if it's not there and I try to scroll up. Probably because it's so ubiquitous for other projects to have the logo on the main page. It's also informative as it directly tells me that this is the landing page.